Cargo.toml

```diff
@@ -21,3 +21,5 @@ codegen-units = 1
 [patch.crates-io]
 zcash_encoding = { path = "components/zcash_encoding" }
 zcash_note_encryption = { path = "components/zcash_note_encryption" }
+schemer = { git = "https://github.com/nuttycom/schemer.git", rev = "b3f7c8a3abb69b67d68258ae5284c5743bdb380b" }
```

Comment:
Should we we relying on a repo fork that is coming from your personal github account?

A denial of service could occur through a failed build if your GitHub user account is deleted or you delete the repo.

Do we have a place we fork things that are kept under the zcash org?

Reply:
We can probably use  "zcash-hackworks" if it's something that's in the works

Reply:
We can fork it either to `zcash` or `zcash-hackworks`. Depending upon whether the original maintainer wants to bother with accepting patches, we might also publish it as our own crate (it's MIT/Apache2 so we just need to attribute properly.)

Reply:
The original maintainer [seems interested in these changes](https://github.com/aschampion/schemer/pull/13#issuecomment-1203774567), so I think it's fine for now to rely on a fork. If we don't want to have that fork be under @nuttycom's repo (though we _did_ do that for temporary `incrementalmerkletree` changes that were merged, due to them referring to an open PR on a fork of our repo), then I'd make it under `zcash-hackworks`, but given that this is pinning a revision rather than a branch, I'm not too concerned (the moment anyone builds this branch locally, they will have a cached copy of the git repo in their `~/.cargo` subdir, so we can recreate it in the event of repo loss).